src/modules/person/domain/personenkontext.service.ts

```diff
@@ -105,4 +105,24 @@ export class PersonenkontextService {
 
         return { ok: true, value: saved };
     }
+
+    public async deletePersonenkontextById(id: string, revision: string): Promise<Result<void, DomainError>> {
+        const personenkontext: Option<PersonenkontextDo<true>> = await this.personenkontextRepo.findById(id);
+
+        if (!personenkontext) {
+            return { ok: false, error: new EntityNotFoundError('Personenkontext', id) };
+        }
+
+        if (personenkontext?.revision !== revision) {
+            return { ok: false, error: new MismatchedRevisionError('Personenkontext') };
+        }
+
+        const deletedRows: number = await this.personenkontextRepo.deleteById(id);
```

Comment:
deletedRows is very "database"-y. I would rather use "deletedPersons" to make the closer connection to the Domain more clear.

Reply:
✅